src/components/MintForm/MintForm.jsx

```diff
@@ -34,6 +34,40 @@ const MintForm = () => {
         e.target.value = '';
     };
 
+    const handleConnectionBetweenBeToFe = async () => {
```

Comment:
We are invoking this function after the data is pinned. No need to  pin again. This function should only send a request to the back-end. Pass the information needed as function params and send it to the back-end

```suggestion
    const sendEvent = async (eventType, nftId, nftContractAddress) => {
```